pkg/upstream/cluster/edf.go

```diff
@@ -18,16 +18,15 @@
 package cluster
 
 import (
-	"sync"
-	"time"
-
 	"mosn.io/mosn/pkg/config/v2"
+	"sync"
 )
 
 type edfScheduler struct {
 	lock        sync.Mutex
 	items       *edfHeap
 	currentTime float64
+	clockIndex  int64
```

Comment:
There is no need to add `Index` suffix, just `clock` is enough

Reply:
ok